src/lang/es.json

```diff
@@ -812,8 +813,18 @@
         "showBalances": "Mostrar saldos",
         "hideBalances": "Ocultar saldos",
         "settings": "Ajustes",
-        "switchTheme": "Cambiar a {theme} Modo",
-        "switchLanguage": "Elige lengua"
-    },
-    "sentText": "Enviado"
+        "switchTheme": "Cambiar al modo {theme}",
+        "switchLanguage": "Elige lenguaje",
+        "selectNode": "@:selectNodeText"
+    },
+    "sentText": "Enviado",
+    "learnMoreText": "Aprende más",
+    "marketMakerCountdown": {
+        "title": "Cuenta Regresiva del Creador de Mercado",
+        "blocks": "bloques",
+        "daysLeft": "días restantes",
+        "txs": "TXs",
+        "volume": "Volumen de Transacciones de  MM",
```

Comment:
It seems that here we have 2 whitespaces before the last word. Could you fix it please?